ext/vulnmdsrc/nvd/nvd.go

```diff
@@ -199,12 +205,16 @@ func getDataFeeds(dataFeedHashes map[string]string, localPath string) (map[strin
 }
 
 func getHashFromMetaURL(metaURL string) (string, error) {
-	r, err := http.Get(metaURL)
+	r, err := httputil.GetWithUserAgent(metaURL)
 	if err != nil {
 		return "", err
 	}
 	defer r.Body.Close()
 
+	if !httputil.Status2xx(r) {
+		return "", errors.New("Unsuccesuful status code: " + string(r.StatusCode))
```

Comment:
`Unsuccessful`

Also having some more context in this message would be nice. I have no idea what request was being tried that returned a non-200 status code.